products/pubsub/api.yaml

```diff
@@ -244,6 +244,15 @@ objects:
               If ttl is not set, the associated resource never expires.
               A duration in seconds with up to nine fractional digits, terminated by 's'.
               Example - "3.5s".
+      - !ruby/object:Api::Type::String
+        name: 'name'
```

Comment:
I think this is supposed to be `name: 'filter'`?

Can you add a test that uses the new field? You can add the field to an existing test here: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/tests/resource_pubsub_subscription_test.go

Reply:
yes, sure! I already fixed the naming.

I will update with the test case. I let you know!

Thank you

Reply:
I've updated the test case 👍